### PR TITLE
better readme, unify constructor API, update changelog

### PR DESCRIPTION
**Pull request by keksa**

### CODE REVIEW

**Review by janprochazkacz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,6 +12,74 @@ The UCRM Plugin SDK can be installed with [Composer](https://getcomposer.org/).
...
+
+    $log->appendLog('Someone tried to access page only for admins with permission to view invoices.');
+
+    die('You\'re not allowed to access this page.');
+}
+
+$log->appendLog('Starting invoice export.');
+
+// Get export format from plugin's configuration.
+// https://github.com/Ubiquiti-App/UCRM-plugins/blob/master/docs/file-structure.md#dataconfigjson
```
Comment (Piskvor) [inline README.md]: This is unclear - are we assuming here that there's an `exportFormat` setting, coming from the manifest?

File: README.md [hunk 2]
```diff
@@ -12,6 +12,74 @@ The UCRM Plugin SDK can be installed with [Composer](https://getcomposer.org/).
...
+// https://ucrm.docs.apiary.io/#reference/invoices/invoicesclientidcreateddatefromcreateddateto/get
+$invoices = $api->get(
+    'invoices',
+    [
+        'order' => 'createdDate',
+        'direction' => 'DESC',
+    ]
+);
+
+foreach ($invoices as $invoice) {
```
Comment (Piskvor) [inline README.md:72]: Perhaps send headers for filetype as well?

**Review by Piskvor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (keksa) [reply README.md]: Yes we are and it's supposed to be unclear as it's just example usage of the SDK. You can get more info at the link in the comment.

File: README.md (hunk 2 above)
Comment (keksa) [reply README.md:72]: no, there is not supposed to be any implementation at all, just example usage of the SDK

**Review by Piskvor — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -12,6 +12,75 @@ The UCRM Plugin SDK can be installed with [Composer](https://getcomposer.org/).
 composer require ubnt/ucrm-plugin-sdk
 ```
 
+## Usage
```
Comment (UBNT-Petr) [inline README.md:25]: show this example below "Available classes"?

Comment (keksa) [reply README.md:25]: sure, why not

**Pull request merged**